js_modules/dagit/packages/core/src/asset-graph/useAssetGraphData.tsx

```diff
@@ -22,22 +22,18 @@ import {
  * uses this option to implement the "3 of 4 repositories" picker.
  */
 export function useAssetGraphData(
-  pipelineSelector: PipelineSelector | null | undefined,
   opsQuery: string,
-  filterNodes?: (assetNode: AssetGraphQuery_assetNodes) => boolean,
+  options: {
+    hideEdgesToNodesOutsideQuery?: boolean;
```

Comment:
Thoughts on making this another option alongside nearest neighbors, upstream, and downstream?

Now that we don't have the global asset graph, it's hard to know what asset nodes are connected to the selected asset. I think it might be nice to add this as an option so that users can see the full view of affected nodes

Reply:
Oh that's a cool idea, we've definitely got the space for it - I'll add a checkbox!